test/HaskellWorks/Data/Network/RangeSpec.hs

```diff
@@ -6,14 +6,21 @@ import HaskellWorks.Hspec.Hedgehog
 import Hedgehog
 import Test.Hspec
 
-import qualified HaskellWorks.Data.Network.Ip.Ip   as V
-import qualified HaskellWorks.Data.Network.Ip.Ipv4 as V4
-import qualified HaskellWorks.Data.Network.Ip.Ipv6 as V6
+import qualified HaskellWorks.Data.Network.Ip.Ip    as V
+import qualified HaskellWorks.Data.Network.Ip.Ipv4  as V4
+import qualified HaskellWorks.Data.Network.Ip.Ipv6  as V6
+import qualified HaskellWorks.Data.Network.Ip.Range as R
 
 {-# ANN module ("HLint: ignore Redundant do"  :: String) #-}
 
 spec :: Spec
 spec = describe "HaskellWorks.Data.Network.RangeSpec" $ do
   describe "Range" $ do
     it "should be tested" $ requireTest $ do
-      True === True
+      R.mergeRanges [] === ([] :: [R.Range Int])
+      R.mergeRanges [R.Range 0 5] === [R.Range 0 5]
+      R.mergeRanges [R.Range 3 5, R.Range 6 7] === [R.Range 3 7]
+      R.mergeRanges [R.Range 0 5, R.Range 7 7] === [R.Range 0 5, R.Range 7 7]
+      R.mergeRanges [R.Range 'a' 'b', R.Range 'c' 'f'] === [R.Range 'a' 'f']
```

Comment:
```suggestion
      R.mergeRanges [R.Range 'a' 'b', R.Range 'c' 'f'] === [R.Range 'a' 'f']
      R.mergeRanges [R.Range 'a' 'a', R.Range 'c' 'f'] === [R.Range 'a' 'f']
      R.mergeRanges [R.Range 'a' 'b', R.Range 'b' 'b'] === [R.Range 'a' 'b']
```

Reply:
`R.mergeRanges [R.Range [] [1], R.Range [1] [2]] === [R.Range [] [2]]`

Reply:
`R.mergeRanges [R.Range (R.Range 0 1) (R.Range 1 1), R.Range (R.Range 2 3) (R.Range 4 6)] === [R.Range (R.Range 0 6)]`

Reply:
No instance for (Enum (R.Range a0))